assemblies/pentaho-solutions/src/main/resources/pentaho-solutions/system/system.properties

```diff
@@ -18,6 +18,12 @@ hideUserHomeFolderOnCreate=false
 # Default folder to open or save file(s) in case user's home folder is hidden
 defaultFolderWhenHomeFolderIsHidden=/public
 
+# Default value to enable migration of Domain objects
+migrate-domains=true
+
```

Comment:
[#6 professionalize code for migrating domain](https://jira.pentaho.com/browse/BACKLOG-36130)